### ISSUE TITLE
Subtitle accessibility with screen readers

### ISSUE THREAD

**Issue by RealAmethyst** [jellyfin/jellyfin-web#7456]
### Description of the bug

Screen readers do not read subtitles on the web-UI. This might be because it's missing an aria live region tag but I could be wrong on that part. Screen readers should be able to automatically pick up subtitles and read them as they are displayed, however this is not the case with Jellyfin. No matter where I focus the screen reader on the page, no speaking subtitles.

### Reproduction steps

Start any screen reader. Comes down to OS Windows: Hold windows key, hold ctrl and press enter.
Mac: Hold command and press F5
Linux: it's usually holding Super, alt and pressing S, although that comes down to distro.
Play something using the UI and listen to the screen reader not speaking subtitles.

### What is the current _bug_ behavior?

No speech when subtitles are displayed.

### What is the expected _correct_ behavior?

Subtitles should speak as they show up on screen and update in realtime, same as shown.

### Jellyfin Server version

10.11.5

### Specify the build version

stable

### Environment

```markdown
OS:windows 11
- Linux Kernel:
- Virtualization:
- Clients:
- Browser:Google chrome
- FFmpeg Version:
- Playback Method:direct play
- Hardware Acceleration:
- CPU Model:
- GPU Model:
- Plugins:
- Reverse Proxy:
- Base URL:
- Networking:
- Jellyfin Data Storage & Filesystem:
- Media Storage & Filesystem:
- External Integrations:
```

### Jellyfin logs

```shell
no relevant log
```

### FFmpeg logs

```shell

```

### Additional information

This is also an issue on Jellyfin clients, not just the server.
